src/status_im/ui2/screens/chat/composer/view.cljs

```diff
@@ -15,7 +15,7 @@
             [status-im.ui2.screens.chat.composer.mentions :as mentions]
             [status-im.ui2.screens.chat.composer.reply :as reply]
             [status-im.ui2.screens.chat.composer.style :as style]
-            [status-im.ui2.screens.chat.photo-selector.view :as photo-selector]
```

Comment:
is it possible to also delete the file "status-im.ui2.screens.chat.photo-selector.view :as photo-selector" as part of this pr?